### DISCUSSION TITLE
Is the v3 migration doc missing custom FactAttribute impementations

### DISCUSSION THREAD

**Discussion by sebastienros** [xunit/xunit#3084] [Question] [answered]
I followed https://xunit.net/docs/getting-started/v3/migration but couldn't find how to migrate custom fact attributes. I have one overriding `Skip` on a custom condition (to run a test only on CI). I found an example in the source that is implementing `IFactAttribute` so followed it.

**Comment by bradwilson** [comment]
I'll make a note to add a doc somewhere. Thanks!

**Reply by bradwilson** [reply, marked as answer]
This is going to be an ongoing labor of love, because of the massive amount of extensibility available in the system, but check out https://xunit.net/docs/getting-started/v3/migration-extensibility and let me know if you think the FactAttribute (and ITestCase/IXunitTestCase) stuff is helpful and/or what other kinds of information I could provide.

Thanks!
